extensions/wikia/AdEngine/AdEngine2ContextService.class.php

```diff
@@ -91,6 +91,14 @@ public function getContext( Title $title, $skinName ) {
 				] ),
 				'forcedProvider' => $wg->AdDriverForcedProvider
 			];
+
+			/**
+			 * $wgAdDriverEnableSourcePointRecovery === false; // disabled on wiki
+			 * $wgAdDriverEnableSourcePointRecovery === true; // enabled on wiki
+			 * $wgAdDriverEnableSourcePointRecovery === null; // don't care - depend on $wgAdDriverSourcePointRecoveryCountries
+			 */
+			$context['opts']['sourcePointRecovery'] = $skinName === 'oasis' ? $wg->AdDriverEnableSourcePointRecovery : false;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the literal "false" boolean value. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1125)